diff --git a/node_modules/react-native-navbar/index.js b/node_modules/react-native-navbar/index.js
index d132036..e830761 100644
--- a/node_modules/react-native-navbar/index.js
+++ b/node_modules/react-native-navbar/index.js
@@ -6,10 +6,13 @@ import {
   Platform,
 } from 'react-native';
 import PropTypes from 'prop-types';
-import ViewPropTypes from './lib';
+
 
 import NavbarButton from './NavbarButton';
 import styles from './styles';
+const ViewPropTypes = typeof document !== 'undefined' || Platform.OS === 'web'
+  ? PropTypes.shape({style: PropTypes.object})
+  : require('react-native').ViewPropTypes || View.propTypes;
 
 const ButtonShape = {
   title: PropTypes.string.isRequired,
